composer.json

```diff
@@ -1,5 +1,5 @@
 {
-    "name": "botme/rights-management-plugin",
+    "name": "attar/rights-management-plugin",
```

Comment:
@ahmed-omani Can you please change it to **3attar/rights-management-plugin**

Reply:
ok

Reply:
> @ahmed-omani Can you please change it to **3attar/rights-management-plugin**

ok